docs/guided-tour.md

```diff
@@ -15,7 +15,7 @@ You can proceed through this tutorial using the [Sandstorm demo server](https://
 
 If you wish, you could take the time now to install Sandstorm on a Linux server of your own. Amazon offers a free-of-cost one year trial where you can use their hosting services for free. To do that, keep reading.
 
-If you don't want to install Sandstorm right now, skip to the **Hands on tour** section.
+If you don't want to install Sandstorm right now, skip to the [**Hands on tour**](https://docs.sandstorm.io/guided-tour/#hands-on-tour) section.
```

Comment:
Please use a link to a `*.md` file not a link to https://docs.sandstorm.io/

I should add a note to README to explain why and how, I suppose, since this is a common issue. Let me know if you need help with that, or if you want me to take care of that for you.